client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/site-setup-design-picker.tsx

```diff
@@ -295,6 +296,11 @@ const SiteSetupDesignPicker: Step = ( { navigation, flow } ) => {
 		window.scrollTo( { top: 0 } );
 	}, [ isForceStaticDesigns, !! selectedDesign ] );
 
+	useEffect( () => {
+		const { innerWidth: width, innerHeight: height } = window;
+		viewport.current = { width, height };
```

Comment:
Do we need to update the viewport when the layout switches to mobile?

Reply:
Nice catch! I've added `isMobile` to the dependency list.